packages/yourloops/pages/profile/profile.tsx

```diff
@@ -500,13 +498,14 @@ const ProfilePage = (props: ProfilePageProps): JSX.Element => {
             >
               {t("button-save")}
             </Button>
-          </div>
-          {UserRoles.caregiver === role ? (
+          </Box>
+
+          {UserRoles.caregiver === role &&
```

Comment:
Why change something that work ?

Reply:
I think it would be better to use React conditionnal rendering ((inline if)) like it's explained in official documentation. It's more readable just for an if condition.